.circleci/config.yml

```diff
@@ -2,23 +2,17 @@ version: 2
 jobs:
   build:
     docker:
-      - image: docker:stable
+      - image: mozilla/cidockerbases:docker-latest
     working_directory: /
+
     steps:
+      - run:
+          name: Host info
+          command: uname -v
+
       - run:
           name: Install essential packages
-          command: |
-            apk update
-            apk add --update --no-cache \
-                    ca-certificates \
-                    build-base \
-                    bash \
-                    make \
-                    git \
-                    openssh \
-                    docker \
-                    py-pip
-            pip install docker-compose
+          command: apt-get install make
```

Comment:
The new image is Ubuntu-based, so we switch to `apt-get`. We need `make` because that's how I've got everything working.